file_store/src/iot_beacon_report.rs

```diff
@@ -144,3 +145,30 @@ impl From<IotBeaconReport> for LoraBeaconReportReqV1 {
         }
     }
 }
+
+impl IotBeaconReport {
```

Comment:
why not `From<IotBeaconReport> for beacon::Beacon {}`?

Reply:
because  the report doesnt have all the data required to generate the beacon.  Requires additional entropy data

Reply:
good point; wasn't paying attention to the other arguments